code/espurna/sensors/BaseSensor.h

```diff
@@ -84,6 +84,9 @@ class BaseSensor {
         // Number of available slots
         unsigned char count() { return _count; }
 
+        // Convert slot # index to a magnitude # index
+        virtual unsigned char local(unsigned char slot) { return 0; }
```

Comment:
naming might be better.
`counts(unsigned char slot)?` (or just `count`), since we mostly repeat what `sensor_magnitude_t::counts()` does. this will accompany already existing `count()`

Reply:
about names/nomenclature, it's hard for me to relate, you overwhelm the whole project, you know better :D

I uploaded your recent changes:
https://github.com/mcspr/espurna/commit/d8196c3a89924dca846c7dc8d9287e7526390b25
works well.

Reply:
thanks : ) just recording some reasoning / warning